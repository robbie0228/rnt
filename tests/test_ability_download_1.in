board
abilities
ability 3 a
ability 3 A
board
move a down
board
abilities
ability 3 a
move A up
move B up
abilities
ability 3 B
board
quit
