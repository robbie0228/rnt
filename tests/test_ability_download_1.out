Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1's Abilities:
1: Link Boost, Unused
2: Firewall, Unused
3: Download, Unused
4: Scan, Unused
5: Polarize, Unused

Cannot use Download on a link that is not your opponent's!

Player 1:
Downloaded: 0D, 1V
Abilities: 4
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
abcSSfgh
...de...
........
........
........
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 1V
Abilities: 4
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE...
.BCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: V1   B: V2   C: V3   D: V4
E: D1   F: D2   G: D3   H: D4

Player 2's Abilities:
1: Link Boost, Unused
2: Firewall, Unused
3: Download, Unused
4: Scan, Unused
5: Polarize, Unused

Invalid move

Player 1's Abilities:
1: Link Boost, Unused
2: Firewall, Unused
3: Download, Used
4: Scan, Unused
5: Polarize, Unused

Ability is unavailable

Player 1:
Downloaded: 0D, 1V
Abilities: 4
a: V1   b: V2   c: V3   d: V4
e: D1   f: D2   g: D3   h: D4
========
.bcSSfgh
...de...
........
........
........
........
.B.DE...
..CSSFGH
========
Player 2:
Downloaded: 0D, 1V
Abilities: 4
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

